src/Cache/CacheContent.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\View\Cache;
+
+use DateInterval;
+use InvalidArgumentException;
+use Yiisoft\Cache\CacheInterface;
+use Yiisoft\Cache\CacheKeyNormalizer;
+use Yiisoft\Cache\Dependency\Dependency;
+
+use function array_merge;
+use function get_class;
+use function gettype;
+use function is_string;
+use function is_object;
+use function sprintf;
+use function strtr;
+
+/**
+ * CacheContent caches content, supports the use of dynamic content {@see DynamicContent} inside cached content.
+ */
+final class CacheContent
+{
+    private string $id;
+    private CacheInterface $cache;
+    private CacheKeyNormalizer $cacheKeyNormalizer;
+
+    /**
+     * @var array<string, DynamicContent>
```

Comment:
```suggestion
     * @var DynamicContent[]
```

Reply:
You can add one object several times and replace placeholders with content several times.
https://github.com/yiisoft/view/pull/159/files#diff-ab378c33bf6ef55354f2cbd7d49a80ec83ef0ff0e3c96b7ff364c58e5b068380R116